docker-compose.yml

```diff
@@ -15,6 +15,23 @@ services:
     networks:
       - shared
 
+  sia:
+    image: nebulouslabs/sia:dev
```

Comment:
This needs to be the stable image. I guess we're gonna switch to a tag as soon as we have one.

Reply:
using a builder here with v1.5.0 branch